staging/nvidia/charts/nvidia-dcgm-exporter/templates/default-counters.yaml

```diff
@@ -0,0 +1,72 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: nvidia-dcgm-exporter-counters
+  namespace: {{ .Release.Namespace }}
+data:
+  default-counters.csv: |
```

Comment:
```suggestion
  default-counters.csv: |
    # These settings are taken from the dcgm-exporter  image
    # https://github.com/NVIDIA/gpu-monitoring-tools/blob/master/etc/dcgm-exporter/default-counters.csv
    #
```

Reply:
thank you @joejulian he did add it so marking this resolved if thats ok.